docker-compose.yml

```diff
@@ -53,6 +73,21 @@ services:
       CONJUR_APPLIANCE_URL: http://conjur:3000
       CONJUR_ACCOUNT: cucumber
 
+  # Spin up container for EE testing
+  test-dap:
+    build:
+      context: .
+      dockerfile: Dockerfile.test
+    volumes:
+      - ./target:/conjurinc/api-java/target
+    entrypoint: sleep
+    command: infinity
+    environment:
+      CONJUR_APPLIANCE_URL: https://cuke-master
+      CONJUR_ACCOUNT: cucumber
+      CONJUR_SSL_CERTIFICATE:
```

Comment:
is this needed? the cert should be set with `keytool` as is done in the oss https container

Reply:
Same comment. See my earlier one in the other CLI invocation.

Reply:
💯